starter-code/scripts/controllers/articleController.js

```diff
@@ -34,7 +37,8 @@
     );
   };
 
-  // COMMENT: What does this method do?  What is it's execution path?
+  // COMMENTed: What does this method do?  What is it's execution path?
+    //based on the seleted option from the navigation menu it finds all articles with the same category value and displays those articles. Upon selection loadByCategory is invoked and then it runs articleController.index when renders the correctly selected articles to the view.
```

Comment:
Same as above. One thing to keep in mind is that articleController.loadByAuthor and articleController.loadByCategory is ITSELF not finding the articles with a given value -- it's calling article.findWhere which does this work. The job of these methods is really calling article.findWhere, passing the appropriate values in, and then defining and providing article.findWhere with a callback function, articleData (that attaches the articles returned from the query onto the ctx object and then calls the next function in the middleware chain).